src/pages/ChatRoomPage.jsx

```diff
@@ -83,10 +105,13 @@ export default function ChatRoomPage() {
     };
 
     (async () => {
-      await getCounterName();
-      await updateReadMessage();
-      await getMessages();
-      updateLastVisited();
+      if (await checkChatRoomUser()) {
+        await getCounterName();
+        await updateReadMessage();
+        await getMessages();
```

Comment:
함수명만 봐서는 직렬적으로 실행될 필요는 없는 함수들인 것 같아요
Promise.all을 사용하여 병렬적으로 API 콜이 일어날 수 있게 해주시면 좋을 것 같습니다!
```suggestion
        await Promise.all([getCounterName(), updateReadMessage(), getMessages()]);
```